### ISSUE TITLE
Interface methods prefixed with 'is' or 'get' cause a compiler error

### ISSUE THREAD

**Issue by Craigus** [ceylon/ceylon-compiler#965]
Using the M4 release, with a fresh Juno/Ceylon IDE install.

The following code gives me an error 
`method does not override or implement a method from a supertype`
`Test.ceylon  /Test/source    line 7  Ceylon Backend Error {code}`

```
shared interface ITest {
    shared default Boolean isFoo() { return false; }
}
shared class CTest() satisfies ITest {
    shared actual Boolean isFoo() {
        return true;
    }
}
```

while this code does NOT produce an error:

```
shared interface ITest {
    shared default Boolean foo() { return false; }
}
shared class CTest() satisfies ITest {
    shared actual Boolean foo() {
        return true;
    }
}
```

Bug also occurs for methods named get_(), but not for methods named set_().
